examples/deploy/deploy.mjs

```diff
@@ -50,19 +62,50 @@ const tg = typegraph({
         t.integer(),
         { wasm: "wasi/rust.wasm", func: "add" }
       ),
+      // Prisma
+      createStudent: prisma.create(student),
+      findManyStudent: prisma.findMany(student),
     }, pub);
   },
 );
 
+const artifactsConfig = {
+  prismaMigration: {
+    action: {
+      create: true,
+      reset: false
+    },
+    migrationDir: "prisma-migrations"
+  }
+};
+const baseUrl = "http://localhost:7890";
+const auth = new BasicAuth("admin", "password");
+
 tgDeploy(tg, {
```

Comment:
What happens with the `meta serialize` command?? :laughing:

Reply:
We can always call `tg.serialize(config?)`

Reply:
Or maybe we can add `tgSerialize(tg, config?)` for consistency..